templates/env.erb

```diff
@@ -1,3 +1,3 @@
 BIND=0.0.0.0
 PORT=<%= scope['katello_devel::rails_port'] %>
-WEBPACK_OPTS='--https --key <%= scope['certs::ca_key'] %> --cert <%= scope['certs::ca_cert'] %> --cacert <%= scope['certs::ca_cert'] %> --host 0.0.0.0 --public <%= @fqdn %>'
+WEBPACK_OPTS='--https --key <%= scope['certs::apache::server_key'] %> --cert <%= scope['certs::apache::server_cert'] %> --cacert <%= scope['certs::apache::default_ca'] %> --host 0.0.0.0 --public <%= scope['certs::apache::hostname'] %>'
```

Comment:
Did you try this? When I tried the installer last week with similar settings, puppet didn't have the values for ::certs::apache::server_cert and server_key for some reason.  Maybe referencing the values in apache.pp fixed it, I didn't have that.

Reply:
I didn't (note the `WIP`). There is a `certs::apache` class definition before any config takes place so in theory ... :)

Reply:
Updated and now with spec tests